pes-events-schema-test.json

```diff
@@ -2,118 +2,154 @@
   "$id": "https://raw.githubusercontent.com/oamg/schema-test/main/pes-events-schema-test.json",
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "Package Evolution Service - Events",
-  "type": "array",
-  "items": [
-    {
-      "type": "object",
+  "$defs": {
```

Comment:
@Wecros  it's better to keep `$defs` section in the bottom of the document than in the top. So people can jump right on the schema and search for defs when they are interested. but it's nitpitck.

Reply:
Moved as requested 👍